dinky-executor/src/main/java/org/dinky/executor/ExecutorFactory.java

```diff
@@ -27,7 +27,9 @@
  **/
 public final class ExecutorFactory {
 
-    public static Executor getExecutor() {
+    private ExecutorFactory() {}
+
+    public static Executor getLocalStreamExecutor() {
         return new LocalStreamExecutor(ExecutorConfig.DEFAULT);
```

Comment:
The default result of getExecutor is LocalStreamExecutor, No need for explicit explanation again.